internal/source/common.go

```diff
@@ -0,0 +1,9 @@
+package source
+
+import (
+	"fmt"
+)
+
+func generateMessage(catalogName string) string {
+	return fmt.Sprintf("Successfully unpacked the %s Bundle", catalogName)
```

Comment:
Nit:
```suggestion
	return fmt.Sprintf("Successfully unpacked the %q catalog", catalogName)
```

Also, tbh I didn't love the PR that added this to rukpak. I wouldn't be opposed to just removing this function and getting rid of (what I think is) an unnecessary abstration.

Reply:
removed in https://github.com/operator-framework/catalogd/pull/65/commits/81397258d6ab05efccacebad76c778798a066bdf